systemtests/src/main/java/com/github/streamshub/systemtests/utils/resourceutils/KafkaCmdUtils.java

```diff
@@ -0,0 +1,144 @@
+package com.github.streamshub.systemtests.utils.resourceutils;
+
+import com.github.streamshub.systemtests.clients.KafkaClients;
+import com.github.streamshub.systemtests.constants.Constants;
+import com.github.streamshub.systemtests.logs.LogWrapper;
+import io.skodjob.testframe.resources.KubeResourceManager;
+import org.apache.logging.log4j.Logger;
+
+public class KafkaCmdUtils {
```

Comment:
Just curious, why use the scripts in the pod rather than the `Admin` Java interface? Is it because you can side-step authentication this way?

Reply:
Basically, but if you feel like admin suits this better i think i could re-work this. Not sure if i can do all of the operations (would have to check to be certain).